test/e2e/deviceplugins_suite_test.go

```diff
@@ -32,21 +33,36 @@ import (
 	_ "github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/qat"
 	_ "github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/sgx"
 	_ "github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/sgxadmissionwebhook"
+	"github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/utils"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/component-base/logs"
 	"k8s.io/component-base/version"
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/config"
-	"k8s.io/kubernetes/test/e2e/framework/kubectl"
+	k "k8s.io/kubernetes/test/e2e/framework/kubectl"
```

Comment:
not sure it's a good idea to use 'k' instead of 'kubelet'. Later is more explicit and readable in the code.